[Term Entry] Python Math Module: math.acos() (#5095)
Codecademy/docs: content/python/concepts/math-module/terms/math-acos/math-acos.md

> ---
> Title: 'math.acos()'
> Description: 'Returns the inverse cosine (in radians) of a value between -1 and 1.'
> Subjects:
>   - 'Computer Science'
>   - 'Data Science'
> Tags:
>   - 'Functions'
>   - 'Arithmetic'
> CatalogContent:
>   - 'learn-python-3'
>   - 'paths/computer-science'
> ---

Reviewer: ```suggestion
---
Title: 'math.acos()'
Description: 'Returns the arc cosine (inverse cosine) of a number.'
Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Arithmetic'
  - 'Functions'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---

```

Reviewer: The description should be the function, and the tags should be alphabetically arranged

> The **`math.acos()`** function returns the inverse cosine (in radians) of a value between -1 and 1.

Reviewer: ```suggestion
In Python, the **`math.acos()`** function returns the arc cosine (inverse cosine) in radians of a value between `-1` and `1`.
```

## Syntax

```pseudo

> math.acos(n)

Reviewer: ```suggestion
math.acos(x)

```

```

> 

Reviewer: ```suggestion

- `x`: A number between `-1` and `1` representing the cosine value for which the arc cosine is to be calculated.

```

Reviewer: The parameter needs to be described, feel free to change the description this is just for reference

> The `math.acos()` function returns a numeric value between 0 and π, which represents the inverse cosine of the value `n`.

Reviewer: ```suggestion
The `math.acos()` function returns a numeric value between 0 and π, which represents the inverse cosine of the value `x`.

```

## Example

> Use `math.acos()` to return the inverse cosine of a numeric value between 0 and π:

Reviewer: ```suggestion
The example below demonstrates the use of `math.acos()` function to return the inverse cosine of a numeric value:

```
